e2e/cypress/integration/account_settings/general/email_spec.js

```diff
@@ -48,4 +60,160 @@ describe('Account Settings -> General -> Email', () => {
         // * Check that the email verification message is not showed.
         cy.get('.announcement-bar').should('not.exist');
     });
+
+    it('MM-T2066 email address required', () => {
+        // # Click "Edit" to the right of "Email"
+        cy.get('#emailEdit').should('be.visible').click();
+
+        // # Save the settings
+        cy.get('#saveSetting').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Check that the correct error message is shown.
+        cy.get('#clientError').should('be.visible').should('have.text', 'Please enter a valid email address');
+    });
+
+    it('MM-T2067 email address already taken error', () => {
+        // # Click "Edit" to the right of "Email"
+        cy.get('#emailEdit').should('be.visible').click();
+
+        // # Type different email
+        cy.get('#primaryEmail').should('be.visible').type(otherUser.email);
+        cy.get('#confirmEmail').should('be.visible').type(otherUser.email);
+        cy.get('#currentPassword').should('be.visible').type(otherUser.password);
+
+        // # Save the settings
+        cy.get('#saveSetting').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Check that the correct error message is shown.
+        cy.get('#serverError').should('be.visible').should('have.text', 'This email is already taken. Please choose another.');
+    });
+
+    it('MM-T2068 email address and confirmation don\'t match', () => {
+        // # Click "Edit" to the right of "Email"
+        cy.get('#emailEdit').should('be.visible').click();
+
+        // # Type some random email
+        cy.get('#primaryEmail').should('be.visible').type('random@example.com');
+        cy.get('#confirmEmail').should('be.visible').clear();
+        cy.get('#currentPassword').should('be.visible').type('randompass');
+
+        // # Save the settings
+        cy.get('#saveSetting').click().wait(TIMEOUTS.HALF_SEC);
+
+        // * Check that the correct error message is shown.
+        cy.get('#clientError').should('be.visible').should('have.text', 'The new emails you entered do not match.');
+    });
+
+    // This test is a combination of 4 sub-tests because they are sub-parts of the same test.
+    // Doing them individually would have a dependency with the previous test.
+    // Hence, a combined single test for everything is better.
+    it('MM-T2069/MM-T2070/MM-T2071/MM-T2072 email address verification', () => {
```

Comment:
Noted. I'll combine the test cases into one, matching the steps here so we can save the result into the TM4J.

Reply:
Combined into [MM-T2069](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2069). Please update the title. Other test cases were archived.
```suggestion
    it('MM-T2069 Email: Can update email address and verify through email notification, () => {
```